clients/venice-client/src/main/java/com/linkedin/venice/fastclient/DispatchingAvroGenericStoreClient.java

```diff
@@ -595,11 +598,19 @@ private long getLatencyInNS(long startTimeStamp) {
     return System.nanoTime() - startTimeStamp;
   }
 
+  private void checkPrerequisites() throws VeniceClientException {
```

Comment:
might rename to `verifyMetadataInitialized`

Reply:
Let's rename it to `verifyMetadataInitialized` to be more clear.